packages/firestore/test/integration/api/database.test.ts

```diff
@@ -140,6 +140,33 @@ apiDescribe('Database', persistence => {
     });
   });
 
+  asyncIt('can delete field using merge', () => {
+    return withTestDb(persistence, db => {
+      const doc = db.doc('rooms/Eros');
```

Comment:
Can you use `withTestDoc(persistence, doc => ... )` instead?  I have a PR I'm going to send out that removes all these hardcoded collection / documents since they could potentially cause flakiness in travis.

Reply:
Sounds good. Done.

Reply:
Nice try.